src/ui/components/ShowMoreCard/index.js

```diff
@@ -55,6 +60,10 @@ export class ShowMoreCardBase extends React.Component {
         <a className="ShowMoreCard-revealMoreLink" href="#show-more"
           onClick={this.expandText}>
           {i18n.gettext('Show more info…')}
```

Comment:
Yes, adding content for screenreader-only should be sufficient. It needs to be a children of the same `<a>` tag to get read out.

I've found a [chrome extension about accessibility developer tools](https://chrome.google.com/webstore/detail/accessibility-developer-t/fpkknkljclfencbdbgkenhalefipecmb) recently, but haven't give it a try. The idea is to be able to inspect the part of the DOM relevant for screenreaders.